Tests/Helper/PathHelperTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Liip\MonitorBundle\Tests\Helper;
+
+use Liip\MonitorBundle\Helper\PathHelper;
+
+class PathHelperTest extends \PHPUnit_Framework_TestCase
+{
+
+    public function testHelperWithSymfony2()
```

Comment:
I would do more of a functional test here .. ie. we do not care about the templating asset helper .. just that things do not break .. since we run the test matrix against all Symfony versions this will ensure that things do not break in the future.